Test/git-issues/git-issue-970b.dfy.expect

```diff
@@ -1,5 +1,5 @@
 git-issue-970b.dfy(7,7): Error: unresolved identifier: Pind
-git-issue-970b.dfy(7,4): Error: The type of the first expression is not a failure type in :- statement
+git-issue-970b.dfy(7,4): Error: The type of the first expression to the right of ':-' could not be determined to be a failure type (got '?')
```

Comment:
Yes, because the symbol is not defined so the type inference assigns a placeholder. I thought of rewriting the message specifically for this use case but I don't think it will be more useful than knowing the symbol does not exists.